src/dotnet/Users.Service/Email/EmailSender.cs

```diff
@@ -0,0 +1,33 @@
+using ActualChat.Users.Module;
+using MailKit.Net.Smtp;
+using MimeKit;
+
+namespace ActualChat.Users.Email;
+
+public interface IEmailSender
```

Comment:
Afair we have SmsGateway, so maybe call then consistently - both Sender or both Gateway...

Reply:
Actually you don't have `SmsGateway`. It's `ITextMessageGateway` and IMO it's too abstract and its name conflicts with a `Gateway` pattern. `SmsSender` is more clear.